var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -167,7 +167,7 @@ class Petsc(Package, CudaPackage, ROCmPackage):
         "memalign",
         default="none",
         description="Specify alignment of allocated arrays",
-        values=("4", "8", "16", "32", "64", "none"),
+        values=("4", "8", "16", "32", "64", "128", "none"),
```

Comment:
current list in petsc library:
```
    valid = ['4', '8', '16', '32', '64', '128', '256', '512', '1024', '2048', '4096', '8192']
```
So perhaps good if this list in spack also matches it.